docs/getting-started.md

```diff
@@ -488,3 +491,121 @@ Now you can create a key, using this security policy, and access your API.
+ show: true
+ version: v2
+```
+
+and then run:
+
+```bash
+kubectl apply -f apiDescriptionHttpbin.yaml
+```
+
+Here we can observe that the API description is tied to the SecurityPolicy
+in the lines
+
+```bash
+ policyRef:
+  name: standard-pol
+```
+
+Furthermore, in the "documentation" field we have a the swagger API definition of httpbin encoded in base64. All documentation descriptions need to be encoded base64.
```

Comment:
It might be better to use backticks (**\`**) to indicate `"documentation"`. So, it will be highlighted by the markdown highlighter.